test/dummy/config/importmap.rb

```diff
@@ -2,19 +2,25 @@
 
 # Pin npm packages by running ./bin/importmap
 
-# link the @turbo-boost/commands build to test/dummy/app/javascript/@turbo-boost
-target = File.expand_path(Rails.root.join("../../app/assets/builds/@turbo-boost"))
-link = File.expand_path(Rails.root.join("app/javascript/@turbo-boost"))
-FileUtils.rm_r link, force: true, verbose: true if File.exist?(link) && !File.symlink?(link)
-FileUtils.ln_s target, link, force: true, verbose: true unless File.exist?(link)
+pin "@hotwired/stimulus", to: "@hotwired--stimulus.js" # @3.2.2
+pin "@hotwired/turbo", to: "@hotwired--turbo.js" # @8.0.2
+pin "@hotwired/turbo-rails", to: "@hotwired--turbo-rails.js" # @8.0.2
+pin "@rails/actioncable", to: "@rails--actioncable.js" # @7.1.3
+pin "@rails/actioncable/src", to: "@rails--actioncable--src.js" # @7.1.3
 
-pin "@hotwired/stimulus", to: "https://ga.jspm.io/npm:@hotwired/stimulus@3.2.2/dist/stimulus.js"
-pin "@hotwired/turbo", to: "https://ga.jspm.io/npm:@hotwired/turbo@7.3.0/dist/turbo.es2017-esm.js"
-pin "@hotwired/turbo-rails", to: "https://ga.jspm.io/npm:@hotwired/turbo-rails@7.3.0/app/javascript/turbo/index.js"
-pin "@rails/actioncable/src", to: "https://ga.jspm.io/npm:@rails/actioncable@7.1.3/src/index.js"
-pin "@turbo-boost/streams", to: "https://ga.jspm.io/npm:@turbo-boost/streams@0.0.8/app/assets/builds/@turbo-boost/streams.js"
-pin "debounced", to: "https://ga.jspm.io/npm:debounced@0.0.5/src/index.js"
+# TODO: I have no idea why the fuck jspm.io refuses to build the latest versions of @turbo-boost libs
+#       You can generate URLs like the one I'm using below here → https://www.jsdelivr.com/github
+pin "@turbo-boost/streams", to: "https://cdn.jsdelivr.net/gh/hopsoft/turbo_boost-streams@v0.1.6/app/assets/builds/%40turbo-boost/streams.js"
```

Comment:
After merging my [PR](https://github.com/hopsoft/turbo_boost-streams/pull/51), you should be able to use:

```ruby
pin "@turbo-boost/streams", to: "@turbo-boost/streams.js"
```